doc/api/api_changes.rst

```diff
@@ -136,23 +104,33 @@ The version of qhull shipped with Matplotlib, which is used for
 Delaunay triangulation, has been updated from version 2012.1 to
 2015.2.
 
+Improved Delaunay triangulations with large offsets
+---------------------------------------------------
 
-Use backports.functools_lru_cache instead of functools32
---------------------------------------------------------
+Delaunay triangulations now deal with large x,y offsets in a better
+way. This can cause minor changes to any triangulations calculated
+using Matplotlib, i.e. any use of `matplotlib.tri.Triangulation` that
+requests that a Delaunay triangulation is calculated, which includes
+`matplotlib.pyplot.tricontour`, `matplotlib.pyplot.tricontourf`,
+`matplotlib.pyplot.tripcolor`, `matplotlib.pyplot.triplot`,
+`mlab.griddata` and `mpl_toolkits.mplot3d.plot_trisurf`.
```

Comment:
s/mlab/matplotlib.mlab/ (not sure if that will work)
s/mplot3d/mplot3d.axes3d.Axes3D/